tests/environment/.env

```diff
@@ -20,7 +20,7 @@ SHARE_TAG=7.2.0
 DIGITAL_WORKSPACE_TAG=2.7.0
 #By default the connector tag is set to the value of `dependency.elasticsearch-shared.version` maven property.
 #Following env variable allow you to override this value. It is useful for testing. It should be left blank on the main branch.
-ES_CONNECTOR_TAG=
+ES_CONNECTOR_TAG=feature_ACS-3634-additional-jdbc-driver-loading-2034
```

Comment:
"It is useful for testing. It should be left blank on the main branch." in a comment right above it, is it a forgotten change? The value also seems kind of weird vs the purpose described in a comment.

Reply:
yup, it is for testing purpose, it has to be there until release of final ES connector as well as this PR should be merge once connector is released.